R/get_raster.R

```diff
@@ -6,7 +6,9 @@
 #' 'DAILY', 'MONTHLY', 'YEARLY'
 #' @param group_by parameter to group by. Can be 'VESSEL_ID', 'FLAG', 'GEARTYPE',
 #'  'FLAGANDGEARTYPE' or 'MMSI'. Optional.
-#' @param filter_by parameter to filter by.
+#' @param filter_by Possible fields to filter by are: flag, geartype and vessel_id.
```

Comment:
I'd add "for AIS based fishing effort" because the filters will change (maybe in a separate function)